scripts/dotnet-cli-build/MsiTargets.cs

```diff
@@ -60,6 +69,20 @@ private static void AcquireWix(BuildTargetContext c)
             ZipFile.ExtractToDirectory(Path.Combine(WixRoot, "WixTools.zip"), WixRoot);
         }
 
+        private static void AcquireCrt(BuildTargetContext c)
+        {
+            if (File.Exists(CrtRedistributablePath))
+            {
+                return;
+            }
+
+            Directory.CreateDirectory(Path.GetDirectoryName(CrtRedistributablePath));
+
+            c.Info($"Downloading VC_redist.{Arch}.{CrtVersion}.exe..");
+
+            DownloadFile($"https://dotnetcli.blob.core.windows.net/build/vc/{Arch}/VC_redist.{Arch}.{CrtVersion}.exe", CrtRedistributablePath);
```

Comment:
I do agree that we should get VCRedist from the MSFt location and not our CLI Blob - otherwise, we will be responsible for updating it and could endup installing a version that is older than what is available on VC Website.

> I believe this means you'd need an internet connection in the case where you didn't have the redist installed and went to install the CLI

How is the CLI acquired when you dont have Internet connection?